Update sharing-workflows-with-your-organization.md (#7301)
github/docs: content/actions/learn-github-actions/sharing-workflows-with-your-organization.md

### Creating a workflow template

Workflow templates can be created by users with write access to the organization's `.github` repository. The templates can then be used by organization members who have permission to create workflows.

{% note %}

> **Note:** Workflow templates can be used to create new workflows in an organizations' public repositories; to use templates to create workflows in private repositories, the organization must be part of an enterprise plan.

Reviewer: ```suggestion

**Note:** Workflow templates can be used to create new workflows in an organization' s public repository; to use templates to create workflows in private repositories, the organization must be part of an enterprise plan.
```

> {% endnote %}

Reviewer: ```suggestion

{% endnote %}
```

### Creating a workflow template

Workflow templates can be created by users with write access to the organization's `.github` repository. The templates can then be used by organization members who have permission to create workflows.

> > **Note:** Workflow templates can be used to create new workflows in an organizations' public repositories; to use templates to create workflows in private repositories, the organization must be part of an enterprise plan.

Reviewer 2: ```suggestion
{% note %}
**Note:** Workflow templates can be used to create new workflows in an organizations' public repositories; to use templates to create workflows in private repositories, the organization must be part of an enterprise plan.
{% endnote %}
```

Reviewer 3: Ok
